framework/src/controller/application.js

```diff
@@ -167,6 +169,8 @@ class Application {
 
 		this.registerModule(ChainModule, {
 			registeredTransactions: this.getTransactions(),
+			lastCommitId: this.config.app.lastCommitId,
+			buildVersion: this.config.app.buildVersion,
```

Comment:
The `.build` file for build version is generated at the time of compiling the binary. We must document it extensively for application development, so they know some examples how we are doing it and how they can do it to identify the build.

Reply:
@nazarhussain where do you think we should put that information? I think lots of things are not documented at the moment. So I'm not sure if it make sense just to highlight that info somewhere. Do we have a parent issue somewhere that we can add this as a child issue, so we can address this later?